Unix/tools/auth_tests_setup.sh

```diff
@@ -37,34 +40,60 @@ do
 done
 
 #
-# if required env variables don't exist, then quit
+# Setup the NTLM auth file for client and server. They must be different files because 
+# if the differeing permissions, as the server file must be owned by root and the client by
+# the user.
+#
+# If there is already a cred file present we defer to it. In configure we possibly copied over
+# a prebuilt cred file for the server from Unix/etc/creds. If such a file exists, it must contain the test
+# credentials.
```

Comment:
I'm not clear on the above. Any "configure"d omiserver.conf would never be copied over for the user, unless of special handing due to --dev or something. Make that clear or this can be very confusing since you're referring to something that is "non-standard" (most users never build, much less build with --dev ... only we do that).